src/test/groovy/com/palantir/gradle/gitversion/GitVersionPluginTests.groovy

```diff
@@ -52,6 +52,27 @@ class GitVersionPluginTests extends Specification {
         buildResult.output.contains('> Cannot find \'.git\' directory')
     }
 
+    def 'git describe works when git repo is multiple levels up' () {
+        given:
+        File rootFolder = temporaryFolder.root
+        projectDir = temporaryFolder.newFolder('level1', 'level2')
+        buildFile = temporaryFolder.newFile('level1/level2/build.gradle')
+        buildFile << '''
+            plugins {
+                id 'com.palantir.git-version'
+            }
+            version gitVersion()
+        '''.stripIndent()
+        Git git = Git.init().setDirectory(rootFolder).call();
+
+        when:
+        // will build the project at projectDir
+        BuildResult buildResult = with('printVersion').build()
+
+        then:
+        buildResult.output.contains(':printVersion\nunspecified\n')
```

Comment:
sorry I didn't catch this earlier, would like to actually have a commit to test we get a real version (as that guarantees there's actually a git folder there)